doc/integrator/docker.rst

```diff
@@ -80,3 +80,53 @@ You can also remove the unused named images, that should be done manually:
 
 * List the images with ``docker images``.
 * Remove the unwanted images with ``docker rmi <image>:<tag>``.
+
+
+Configure project for OpenShift
+-------------------------------
+
+This section is specific to projects using OpenShift for deployment, relying
+on Travis to build Docker images that are then pushed to Docker Hub.
+
+Edit file `.travis.yml` and update the environment variables:
+
+.. prompt:: bash
+
+    env:
+      global:
+        - PROJECT=<package name>
+        - OPENSHIFT_PROJECT=gs-gmf-<package name>
+        - HELM_RELEASE_NAMES=int,prod
+        - OPENSHIFT_URL=https://api.openshift-ch-1.camptocamp.com/
+        - DOCKER_USERNAME=<FIXME>
```

Comment:
OK to publish the actual username here?

Reply:
It's already in the scaffolds: https://github.com/camptocamp/c2cgeoportal/blob/2.4/geoportal/c2cgeoportal_geoportal/scaffolds/create/%2Bdot%2Btravis.yml_tmpl even though only commented out. I guess we can leave it as it.

Do you think it is OK if I actually replace ``<FIXME>`` by the real username C2C uses? Or would it be a security issue?

Reply:
No, he is already visible in the demo project :-)